html/html-and-the-web_personal-website/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Janosch Schilling</title>
+  </head>
+  <body>
+    <header>
+      <h1>Janosch Schilling</h1>
+      <nav></nav>
+    </header>
+    <section>
+      <img src="./assets/IMG_8457.jpg" alt="Janosch Schilling" />
+      <p>Hey this is me Janosch, here you see me in a coding bootcamp</p>
+    </section>
+    <aside>
+      <h3>I like</h3>
+      <ul>
+        item 1
+      </ul>
+    </aside>
+    <section>
+      <h2>Contact me</h2>
+      <h4>Phone:</h4>
+      <a href="1234567">1234567</a>
```

Comment:
link does not bring to an actual website

Reply:
it's a placeholder for now @tonelloalan